charts/coh-cor/requirements.yaml

```diff
@@ -0,0 +1,7 @@
+dependencies:
+  - name: postgresql
+    version: ~2.6.5
+    repository: '@stable'
+  - name: java
+    version: 0.0.11
```

Comment:
you need to update this to ~0.0.12 .... this stops the process being run as root.

Reply:
Thanks, done it.